[2024-06-15T21:02:49.100+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-15T21:02:49.138+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ran_info_api.format_response manual__2024-06-15T20:56:14.861100+00:00 [queued]>
[2024-06-15T21:02:49.152+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ran_info_api.format_response manual__2024-06-15T20:56:14.861100+00:00 [queued]>
[2024-06-15T21:02:49.154+0000] {taskinstance.py:2306} INFO - Starting attempt 5 of 6
[2024-06-15T21:02:49.270+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): format_response> on 2024-06-15 20:56:14.861100+00:00
[2024-06-15T21:02:49.284+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=7532) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-15T21:02:49.288+0000] {standard_task_runner.py:63} INFO - Started process 7536 to run task
[2024-06-15T21:02:49.284+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ran_info_api', 'format_response', 'manual__2024-06-15T20:56:14.861100+00:00', '--job-id', '71', '--raw', '--subdir', 'DAGS_FOLDER/rand_request_api.py', '--cfg-path', '/tmp/tmp875flvl3']
[2024-06-15T21:02:49.290+0000] {standard_task_runner.py:91} INFO - Job 71: Subtask format_response
[2024-06-15T21:02:49.443+0000] {task_command.py:426} INFO - Running <TaskInstance: ran_info_api.format_response manual__2024-06-15T20:56:14.861100+00:00 [running]> on host d4d3ae2e009b
[2024-06-15T21:02:49.701+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='protectorate' AIRFLOW_CTX_DAG_ID='ran_info_api' AIRFLOW_CTX_TASK_ID='format_response' AIRFLOW_CTX_EXECUTION_DATE='2024-06-15T20:56:14.861100+00:00' AIRFLOW_CTX_TRY_NUMBER='5' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-15T20:56:14.861100+00:00'
[2024-06-15T21:02:49.703+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-15T21:02:49.724+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-15T21:02:49.725+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/rand_request_api.py", line 17, in format_response
    result = res['results'][0]
             ~~~^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
[2024-06-15T21:02:49.742+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=ran_info_api, task_id=format_response, run_id=manual__2024-06-15T20:56:14.861100+00:00, execution_date=20240615T205614, start_date=20240615T210249, end_date=20240615T210249
[2024-06-15T21:02:49.869+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 71 for task format_response (string indices must be integers, not 'str'; 7536)
[2024-06-15T21:02:49.906+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-15T21:02:49.945+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-15T21:02:50.069+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
